exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -1,4 +1,6 @@
-# Basics
+# Introduction
+
+## Basics
```

Comment:
So with this addition it makes the website look like this:

![Screenshot from 2021-05-16 11-53-53](https://user-images.githubusercontent.com/8953691/118407346-a86e6700-b63d-11eb-98dc-0605508c726a.png)

I don't think 'Basics' makes sense as the first h3 to describe the content.  If anything we should, depending on the exercise and introduction associated, make this:

```suggestion
## Basics

### Variables
```

or leave the concept name out and just use heading to describe the content

Reply:
I like the suggestion of adding a "Variables" heading 👍